src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -92,12 +89,13 @@ public void setNumberOfEggs(int numberOfEggs) {
     }
 
     public int getAgeOfQueen() {
-        return ageOfQueen;
-    }
+       return ageOfQueen=getQueen().getAgeOfQueen();
+   }
 
     public void setAgeOfQueen(int ageOfQueen) {
```

Comment:
`ageOfQueen` is not used, you can remove it

Reply:
done